src/Uno.UI/UI/Xaml/Controls/Repeater/StackLayout.cs

```diff
@@ -0,0 +1,354 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections.Specialized;
+using Windows.Foundation;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using static Microsoft.UI.Xaml.Controls._Tracing;
+#if NETSTANDARD2_0
+using _Double = Uno.UI.LayoutHelper;
+#else
+using _Double = System.Double;
+#endif
+
+namespace Microsoft.UI.Xaml.Controls
+{
+	public partial class StackLayout : VirtualizingLayout, IFlowLayoutAlgorithmDelegates
+	{
+		private double m_itemSpacing;
+
+		#region IFlowLayout
+
+		public StackLayout()
+		{
+			//__RP_Marker_ClassById(RuntimeProfiler.ProfId_StackLayout);
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5863680)